woocommerce/woocommerce-gutenberg-products-block
# Remove `fromEntriesPolyfill`

This PR will remove the `fromEntriesPolyfill` we have - this is no longer required as `Object.fromEntries` is [supported by all modern browsers](https://caniuse.com/mdn-javascript_builtins_object_fromentries), and given we no longer support IE11 there is no need to keep it around.

### Testing

#### Automated Tests
### Manual Testing

How to test the changes in this Pull Request:

1. Ensure nothing breaks when checking out. Change shipping rates several times.
2. Use WooCommerce Subscriptions to check out with multiple shipping packages and ensure you can select different shipping rates without issues.

### User Facing Testing
These are steps for user testing (where "user" is someone interacting with this change that is not editing any code).
